src/dao/api.dao.ts

```diff
@@ -335,6 +345,73 @@ export class NodeDao {
     }
   };
 
+  async getFeedEntriesV3(limit: number, cursor: string | null, query: string | null) {
```

Comment:
ok good so we can compare with previous one, did you do any tests?

Reply:
Yes, all tested and it's working as expected